packages/backend/src/lib/backends/github.ts

```diff
@@ -482,51 +482,65 @@ export async function addUserToOrganization(token: string, org: string, username
 }
 
 export async function getCommitList(owner: string, repo: string): Promise<GitHubRepositoryHistoryEdge[]> {
+  let hasNextPage = true;
+  let endCursor: string | undefined;
   const edges: GitHubRepositoryHistoryEdge[] = [];
-  try {
-    const { repository }: { repository: GitHubRepository } = await makeGraphql()({
-      query: `query commitList($owner: String!, $repo: String!) {
-      repository(owner: $owner, name: $repo) {
-        id
-        name
-        defaultBranchRef {
-          target {
-            ... on Commit {
-              history(first:100) {
-                edges {
-                  node {
-                    ... on Commit {
-                      message
-                      author {
-                        name
-                        user {
-                          login
-                          name
+
+  while (hasNextPage === true) {
+    try {
+      const { repository }: { repository: GitHubRepository } = await makeGraphql()({
+        query: `query commitList($owner: String!, $repo: String!${endCursor ? ', $after: String!' : ''}) {
+          repository(owner: $owner, name: $repo) {
+            id
+            name
+            defaultBranchRef {
+              target {
+                ... on Commit {
+                  history(first:100, ${endCursor ? ', after: $after' : ''}) {
```

Comment:
Do you have an extra `,` here?  I think the comma should only be included if endCursor is set

Reply:
You're right, let me fix it